drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-api/pom.xml

```diff
@@ -65,6 +65,13 @@
       <artifactId>drools-scenario-simulation-api</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.jacoco</groupId>
```

Comment:
what is the reason for declaring this dependency here?

Reply:
@rsynek 
Without that I had classnotfound exception.

Reply:
Have you found out why is the dependency required here? I should be already on classpath due to [1] and I can't see a reason why it needs to be declared here, especially outside any profile.

[1] https://github.com/kiegroup/drools-wb/pull/1217/files#diff-600376dffeb79835ede4a0b285078036R310

Reply:
@rsynek 
Removed